src/video/vita/SDL_vitatouch.c

```diff
@@ -62,8 +63,10 @@ VITA_InitTouch(void)
 	}
 
 	// Support passing both front and back touch devices in events
-	SDL_AddTouch((SDL_TouchID)0, "Front");
-	SDL_AddTouch((SDL_TouchID)1, "Back");
+	SDL_AddTouch((SDL_TouchID)0, SDL_TOUCH_DEVICE_DIRECT, "Front");
+	SDL_AddTouch((SDL_TouchID)1, SDL_TOUCH_DEVICE_DIRECT, "Back");
+
+	SDL_SetHintWithPriority(SDL_HINT_TOUCH_MOUSE_EVENTS, "0", SDL_HINT_DEFAULT);
```

Comment:
This is a change @rsn8887 suggested since he already knew about the same issue on the Switch port: https://github.com/devkitPro/SDL/pull/56

Thank you!! 😄